go.mod

```diff
@@ -3,10 +3,11 @@ module github.com/livepeer/lpms
 go 1.12
 
 require (
-	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
+	github.com/golang/glog v0.0.0-20210429001901-424d2337a529
 	github.com/golang/protobuf v1.5.2
+	github.com/livepeer/go-livepeer v0.5.20 // indirect
```

Comment:
Seems like a cyclic dependency on `go-livepeer` was accidentally added here.
We should remove this, and run `go mod tidy` again to clean up `go.sum` below.